CMakeLists.txt

```diff
@@ -68,7 +68,9 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 string(REPLACE "-O3" "-O2" CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE}")
 
 # Enable -Wconversion.
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wconversion")
+if(NOT MSVC)
+  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wconversion")
```

Comment:
It might be because we whitelist which files get Wconversion applied. Might as well leave it, we can revisit after the whitelist is removed.